tasks/__init__.py

```diff
@@ -426,28 +426,42 @@ def flake():
 
 
 @task(aliases=['req'])
-def requirements(addons=False, release=False, dev=False, metrics=False):
+def requirements(base=False, addons=False, release=False, dev=False, metrics=False, quick=False):
     """Install python dependencies.
 
     Examples:
+        inv requirements
+        inv requirements --quick
 
-        inv requirements --dev
-        inv requirements --addons
-        inv requirements --release
-        inv requirements --metrics
+    Quick requirements are, in order, addons, dev and the base requirements. They should do for day-to-day development
+    work so you get everything that someone might have added that would keep you from running the software properly.
+
+    By default, base requirements will run. However, if any set of addons, release, dev, or metrics are chosen, base
```

Comment:
> By default, base requirements will run.
Except `base=False`, and if I want to include requirements for addons, dev, or metrics the base requirements are not installed.

Maybe I'm not  reading the code correctly, but for:
> Or you could do invoke requirements --quick which will do, in order, addons, dev, and base.

I'd think the else branch on L455 would get hit with: `dev=True, metrics=True, base=False`